lib/smart_proxy_openscap/openscap_api.rb

```diff
@@ -42,5 +43,16 @@ class Api < ::Sinatra::Base
 
       {"created" => true}.to_json
     end
+
+    get "/policies/:policy_id/content" do
+      content_type 'application/xml'
+      begin
+        Proxy::OpenSCAP::get_policy_content(params[:policy_id])
+      rescue OpenSCAPException => e
+        log_halt e.http_code, "Error fetching xml file: #{e.message}"
+      rescue StandardError => e
+        log_halt 500, "Error occurred: #{e.message}"
+      end
+    end
   end
 end
```

Comment:
Also, while not subject of this PR, but it caught my attention -- looks strange that an http-related exception can be thrown by a directory creating method: https://github.com/shlomizadok/smart_proxy_openscap/blob/proxy_scap_content/lib/smart_proxy_openscap/openscap_api.rb#L30